cache/core/core_test.go

```diff
@@ -0,0 +1,156 @@
+package core_test
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/fastly/compute-sdk-go/cache/core"
+)
+
+func ExampleLookup() {
+	// f is a core.Found value, representing a found cache item.
+	// core.ErrNotFound is returned if the item is not cached.
+	f, err := core.Lookup([]byte("my_key"), core.LookupOptions{})
+	if err != nil {
+		panic(err)
+	}
+	defer f.Body.Close()
+
+	// The contents of the cached item are in the Found's Body field.
+	cachedStr, err := io.ReadAll(f.Body)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Printf("The cached value was: %s", cachedStr)
+}
+
+func ExampleInsert() {
+	const (
+		key      = "my_key"
+		contents = "my cached object"
+	)
+
+	// w is a core.WriteCloseAbandoner, a superset of io.CloseWriter.
```

Comment:
`s/io.CloseWriter/io.WriteCloser/`

Reply:
Thanks.  Fixed this in a force-push, 8d4d09f.  https://github.com/fastly/compute-sdk-go/compare/3178ed9e2ef70d880148a7fe0bc4520645ade5a4..8d4d09ffd8cec0b325d43866ffeae113400dce44